backend/corpora/common/corpora_orm.py

```diff
@@ -14,17 +14,46 @@
     String,
     text,
 )
+from sqlalchemy.ext.declarative import DeclarativeMeta
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship, sessionmaker
 
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-from common.corpora_config import CorporaDbConfig
+from .corpora_config import CorporaDbConfig
 
-Base = declarative_base()
-deployment_stage = os.environ["DEPLOYMENT_STAGE"]
 
+class TransformingBase(object):
+    """
+    Add functionality to transform a Base object, and recursively transform its linked entities.
+    """
+
+    RELATIONSHIPS_TO_DICT = True  # Set to True if entities with a relationship to this should be converted to a dict.
+
+    def __iter__(self):
+        return self.to_dict().iteritems()
+
+    def to_dict(self, relation=None, backref=None):
```

Comment:
Can we use slightly more descriptive argument names here please? The idea behind descriptive names is that in one read, I can tell what the argument is supposed to do. I was really confused for a bit because I thought `relation` represented the relationships themselves and got confused by the boolean nature of it. I'd have preferred something more descriptive, even if it's longer.

Reply:
That's fair, the variable was recycled and changed types.

Reply:
fixed